docs/reference/commandline/save.md

```diff
@@ -12,7 +12,7 @@ parent = "smn_cli"
 
     Usage: docker save [OPTIONS] IMAGE [IMAGE...]
 
-    Save an image(s) to a tar archive (streamed to STDOUT by default)
+    Save images to a tar archive (streamed to STDOUT by default)
```

Comment:
Perhaps `Save image(s)` to more clearly indicate that one or more images are supported. Or (to be consistent with [`docker rm`](https://docs.docker.com/engine/reference/commandline/rm/); "Save one or more images to...". wdyt?